ledger/participant-state/kvutils/BUILD.bazel

```diff
@@ -208,7 +208,6 @@ da_java_proto_library(
         ) if not is_windows else None,
     )
     for (name, port, environment_variable, checker) in [
```

Comment:
This list comprehension is now overkill. Just inline the values.

Reply:
Actually, it avoids duplicating the port; not a bazel expert, is there a better syntax to define that bunch of constants?

Reply:
We duplicate it everywhere else, but if you want to avoid that, there's nothing wrong with adding a constant above. Caps are traditional, but not mandatory.

```bazel
REFERENCE_LEDGER_EXPORT_PORT = 65101
```

Reply:
Done, thanks!